includes/class-bsr-ajax.php

```diff
@@ -116,7 +116,7 @@ public function process_search_replace() {
 
 			// Build the arguements for this run.
 			$args = array(
-				'select_tables' 	=> isset( $args['select_tables'] ) ? $args['select_tables'] : array(),
+				'select_tables' 	=> isset( $args['select_tables'] ) ? $this->escape_selected_tables($args['select_tables']) : array(),
```

Comment:
This function does not do anything extra to prevent the reported sql injection as it also does not escape backticks (\`):
![image](https://user-images.githubusercontent.com/5002624/180406778-d3e26d1f-ad05-48c2-81a1-e29888702f5b.png)